internal/exec/stages/disks/luks.go

```diff
@@ -224,3 +239,43 @@ func (s *stage) createLuks(config types.Config) error {
 
 	return nil
 }
+
+func (s *stage) checkLuksDeviceExists(luks types.Luks, keyFilePath string) (bool, error) {
+	if _, err := s.Logger.LogCmd(
+		exec.Command(distro.CryptsetupCmd(), "isLuks", *luks.Device),
```

Comment:
Can we add `--type luks2` here? IOW, we don't consider the pre-existing LUKS device equivalent if it matches in all respect but is LUKS1. Clevis at least works differently on LUKS1 vs LUKS2. If we can, it'd be easier to only have to support the latter.